src/test.ts

```diff
@@ -1,20 +1,25 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
-
+/**
+ * IMPORTANT
+ * Keep the imports zone.js first in order, otherwise the karma runner breaks.
+ * Required order is: 'zone.js', 'zone.js/testing' ...
+ */
```

Comment:
Add `// tslint:disable:ordered-imports` to prevent tslint reorder
```suggestion
 */
 // tslint:disable:ordered-imports
```